CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ Summary
  * Enh #1436: Add code to detect old cache directories
  * Enh #1439: Improve cancellation logic
  * Enh #11: Add the `diff` command
+ * Enh #1522: Add support for TLS client certificate authentication.
```

Comment:
I'm sorry about that, but can you please remove the changes from `CHANGELOG.md`, it'll be generated only shortly before the release. We've built our own changelog generator (called [calens](https://github.com/restic/calens)) and now you need to create a file in a subdir of `changelog/`.

I've also updated the PR template.